guides/git-usage.md

```diff
@@ -0,0 +1,72 @@
+---
+category: Tools
+expires: 2018-05-31
+title: Git Usage
```

Comment:
Can you move this to be a `h1` in the body of the document, like the rest of the `.md` files in this repo?